projects/rift-finance/addresses.js

```diff
@@ -1,6 +1,12 @@
 module.exports = {
   aurora: {
-    core: "0x40A01A4064b690cA33FA52d315ec02015eF5287E",
+    coreAddress: "0x40A01A4064b690cA33FA52d315ec02015eF5287E",
```

Comment:
`addresses.js` isn't super accurate anymore, let's call it `networks.js` or something more descriptive